CHANGELOG.md

```diff
@@ -3,11 +3,30 @@
 ### Added
 
 - Add readiness probe to etcd pod. The readiness state will be reflected on `status.members.ready/unready`.
+- TLS etcd cluster support in backup/restore-operator.
+- Add spec validation in restore operator.
+- Add BackupStorageType to EtcdRestore.RestoreSpec to indicate type of the backup storage which is used as RestoreSource.
+- Add EtcdClusterRef to EtcdRestore.RestoreSpec to reference an EtcdCluster resource which should have "Failed" status.phase and to be restored.
+- Add create-crd flag to etcd backup operator allowing user to disable automatic backup crd creation.
+- Add create-crd flag to etcd restore operator allowing user to disable automatic restore crd creation.
+- Add EtcdVersion and EtcdRevision to BackupStatus.
+- Add validation of BackupStorageType in restore operator.
+- Add run e2e tests from container.
+
 
 ### Changed
 
+- Provide scripts to run e2e tests inside a pod.
+- Use default etcd version when creating NewEtcdRestore.
```

Comment:
Delete 
`Provide scripts to run e2e tests inside a pod.` and 
`Use default etcd version when creating NewEtcdRestore.`